types/paddle-js-browser/index.d.ts

```diff
@@ -0,0 +1,231 @@
+// Type definitions for non-npm package Paddle.js 2.0
+// Project: https://developer.paddle.com
+// Definitions by: Christopher Atlan <https://github.com/catlan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+	interface Window {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/paddle-js-browser/index.d.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.